docs/source/getting-started.rst

```diff
@@ -4,33 +4,51 @@ LLRI stands for "Low Level Rendering Interface" and refers to the LLRI API. The
 
 Platforms
 ------------
-LLRI is built with C++17, and currently only builds for Windows. It supports the following graphics APIs:
+LLRI is built with C++17, and supports the following configurations:
 
-* Vulkan
-* DirectX 12
+============================	==========	====================
+Operating System    			Compiler	Implementation(s)
+============================	==========	====================
+Windows 10 (x64)				MSVC        Vulkan / DirectX 12
+Windows 10 (x64)				LLVM Clang	Vulkan / DirectX 12
+MacOS 11 Big Sur				G++			Vulkan (MoltenVK)
+MacOS 11 Big Sur				Clang++		Vulkan (MoltenVK)
+Ubuntu 20.0.4					G++			Vulkan
+Ubuntu 20.0.4					Clang++		Vulkan
+============================	==========	====================
```

Comment:
These tables don't seem to function right?

Reply:
Hmm seems like my tabs misbehaved

Reply:
Should be fixed now